_posts/2022-11-07-relicensing.md

```diff
@@ -0,0 +1,200 @@
+
+
+## Rationale for license change
+
+
+### Why not GPL any more?
+
+Since its initial release in 2008, MDAnalysis has grown from a small
+Python package used by a handful of enthusiastic graduate students and
+postdocs to a mature library that is used by thousands of researchers
+in the molecular sciences. The MDAnalysis library was published under
+an open source license from the start so that anyone could freely use
+it, contribute to it, and build on it. We chose the GNU Public License
+(GPL) for this purpose. Thanks to the "copy-left" portion of the GPL,
+which requires anyone using MDAnalysis in their own code to also adopt
+the GPL for their code, contributors could feel that any time and work
+that they donated and invested into MDAnalysis would not end up
+contributing to software under non open-source licensing.
+
+However, the GPL has also created barriers to adoption of MDAnalysis.
+It has, under our (and many other's) interpretation of the GPL license,
+prevented developers from making their own code available under non-GPL
+licenses. Ultimately, it is the MDAnalysis core team's view that we do
+not want to dictate how our developers and users should license their
+code, but we also wish to ensure that work on the MDAnalysis library
+remains open and free.
+
+Doing so would ultimately benefit the MDAnalysis community, increasing
+the number of codes which can use MDAnalysis, and enabling users in
+corporate environments to use the library with more certainty. The
+reduced licensing complexity also paves the way for our proposed
+[MDAKit ecosystem]({{ site.baseurl }}{% post_url 2022-08-24-mdakits-intro %}).
+
+
+### Why now the LGPL?
+
+We therefore propose to undergo the process of **relicensing MDAnalysis
+under the [Lesser GNU Public License v3 (or any later versions)][LGPL]**.
+This open source license fulfills a number of important requirements for us:
+
+1. Downstream codes are able to freely import or link to MDAnalysis
+   library components without impacting the license choice of the
+   downstream code.
+   
+2. Downstream codes are able to use and subclass any MDAnalysis components
+   under its application programming interface (classes, methods, and
+   data objects), without impacting the license choice of the
+   downstream code.
+   
+3. Codes which either copy or extend the MDAnalysis library should
+   fall under the copyleft license requirements of the MDAnalysis
+   library license.
+   
+Thus, it is our view that the LGPL gives people the freedom to choose
+any license for their own code that *makes use* of the MDAnalysis library
+as a whole (namely ``import MDAnalysis`` or subclassing). This includes
+closed / commercial licenses (although we encourage the use of open
+source licenses). However, one would not be able to just take parts of
+the MDAnalysis code and put it into another piece of code unless the
+other code is then *also* licensed under a compatible copyleft license
```

Comment:
Wording here of "compatible copyleft license" is also some careful wording since GPLv2 wouldn't be compatible, etc...